contributing-docs/03_contributors_quick_start.rst

```diff
@@ -462,6 +462,8 @@ see in CI in your local environment.
                 --role Admin \
                 --email admin@example.org
 
+.. note::
+    ``airflow users`` command is only available when FAB auth manager is enabled.
```

Comment:
Users might not be aware of the concept of auth managers. I think we should add a link to https://airflow.apache.org/docs/apache-airflow-providers-fab/stable/auth-manager/index.html. Maybe make "FAB auth manager" clickable towards https://airflow.apache.org/docs/apache-airflow-providers-fab/stable/auth-manager/index.html?

That applies to your other changes as well

Reply:
Done!